packages/web-components/fast-foundation/src/avatar/avatar.template.ts

```diff
@@ -10,16 +10,13 @@ export function avatarTemplate(
 ): ElementViewTemplate<FASTAvatar> {
     return html<FASTAvatar>`
     <div
-        class="backplate ${x => x.shape}"
+        class="backplate"
         part="backplate"
-        style="${x =>
-            x.fill ? `background-color: var(--avatar-fill-${x.fill});` : void 0}"
     >
```

Comment:
With this change, I wonder - do we need the wrapping div here or can this just be the host?

Reply:
Good question. I'm still finishing up that full component structure audit, and I believe there will be a few key updates we can make for consistency. The "base" element is one of those. When do we use a template or a container. etc.
I mentioned I would probably follow Badge. I see that has a wrapper "control" div, which given the cleanup I just mentioned, we may not actually need. In the case of Avatar it looks like this "backplate" div is important as it provides the only way to style the shape and color of the actual avatar portion separate from the slotted badge that may overlay or appear positioned relative to the backplate somehow. I suggest we keep this.

Reply:
My .02 would be we use template when we need to bind data to the host. We rely on the *host* for styling if we can, we add an element for structure if we can't. This is a good example of the latter w/o a template (as we aren't binding data).